### DISCUSSION TITLE
driver.open_new_window() and driver.open_new_tab() dont work anymore?

### DISCUSSION THREAD

**Discussion by RickVK04** [seleniumbase/SeleniumBase#2266] [Q&A] [answered]
Hi there,

Whenever I use one of these two I get the following error, which I didn't get before:
AttributeError: 'Chrome' object has no attribute 'open_new_window'. Anyone know how to solve this?
I'm running this in UC mode, although I don't think that makes a difference. I updated SeleniumBase and this also didn't help.

**Comment by mdmintz** [comment, marked as answer]
`open_new_window()` and `open_new_tab()` are SeleniumBase BaseCase methods, not `driver` methods. Here's an example of it:

[SeleniumBase/examples/test_window_switching.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_window_switching.py)

You may want to look at [SeleniumBase/help_docs/syntax_formats.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md) to learn the difference.

**Reply by RickVK04** [reply]
Okay. So I read the docs but still wonder if it's possible then to use UC mode and BaseCase methods. Is that possible?

**Reply by mdmintz** [reply]
Yes, ALL the current UC Mode examples in the SeleniumBase `examples` folder are using a format that supports `BaseCase` methods (via `BaseCase` inheritance, or the `SB()` manager):
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_uc_mode.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/verify_undetected.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/uc_cdp_events.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_turnstile.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_form_turnstile.py

**Reply by HumbleExit** [reply]
Almost asked the same question. This helped a lot!
